CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.7.1 (2019-11-27)
+
+- Fixed the repeater item component to not steal focus from focasable child elements when they are activated. [#96](https://github.com/blackbaud/skyux-lists/pull/96)
```

Comment:
i think maybe that should be "focusable" not "focasable"

Reply:
Good catch. Typing faster than by brain is working apparently. Fixed.